ArcGISRuntimeSDKQt_CppSamples/Scenes/Animate3DSymbols/Animate3DSymbols.h

```diff
@@ -34,37 +34,35 @@ namespace Esri
   }
 }
 
-class QAbstractItemModel;
-
-#include "Point.h"
+class QAbstractListModel;
+class MissionData;
 
 #include <QQuickItem>
 #include <QString>
 
-#include <cmath>
 #include <memory>
 
 class Animate3DSymbols : public QQuickItem
 {
-  Q_OBJECT
-
   Q_PROPERTY(bool missionReady READ missionReady NOTIFY missionReadyChanged)
   Q_PROPERTY(int missionSize READ missionSize NOTIFY missionSizeChanged)
-  Q_PROPERTY(int missionFrame READ missionFrame WRITE setMissionFrame)
+  Q_PROPERTY(int missionFrame READ missionFrame WRITE setMissionFrame NOTIFY missionFrameChanged)
+  Q_PROPERTY(QAbstractListModel* missionModel READ missionsModel CONSTANT)
   Q_PROPERTY(double minZoom READ minZoom NOTIFY minZoomChanged)
-  Q_PROPERTY(double zoom READ zoom WRITE setZoom)
-  Q_PROPERTY(double angle READ angle WRITE setAngle)
+  Q_PROPERTY(double zoom READ zoom WRITE setZoom NOTIFY zoomChanged)
+  Q_PROPERTY(double angle READ angle WRITE setAngle NOTIFY angleChanged)
+  Q_OBJECT
 
 public:
-  explicit Animate3DSymbols(QQuickItem* parent = nullptr);
+  Animate3DSymbols(QQuickItem* parent = nullptr);
   ~Animate3DSymbols();
 
-  void componentComplete() Q_DECL_OVERRIDE;
+  void componentComplete() override;
   static void init();
 
   Q_INVOKABLE void animate();
   Q_INVOKABLE void changeMission(const QString& missionName);
-  Q_INVOKABLE QAbstractItemModel* missionsModel() {return m_missionsModel;}
+  QAbstractListModel* missionsModel() {return m_missionsModel;}
```

Comment:
should be const and have the same name as the property. Also probably move the implementation to the cpp